src/marshmallow/fields.py

```diff
@@ -1114,15 +1114,62 @@ def _make_object_from_format(value, data_format):
         return dt.datetime.strptime(value, data_format)
 
 
-class LocalDateTime(DateTime):
-    """A formatted datetime string in localized time, relative to UTC.
+class NaiveDateTime(DateTime):
+    """A formatted naive datetime string.
+
+    :param str format: See :class:`DateTime`.
+    :param timezone timezone: Used on deserialization. If None, aware datetimes
+        are rejected. If not None, aware datetimes are converted to this
+        timezone before their timezone information is removed.
+    :param kwargs: The same keyword arguments that :class:`Field` receives.
+    """
+
+    AWARENESS = "naive"
 
-        ex. ``"Sun, 10 Nov 2013 08:23:45 -0600"``
+    def __init__(self, format=None, timezone=None, **kwargs):
+        super().__init__(format=format, **kwargs)
+        self._timezone = timezone
```

Comment:
Why the underscore?

Reply:
Well. It is related to https://github.com/marshmallow-code/marshmallow/issues/1281. But this has not been settled yet, so let's keep everything not underscored for now, for consistency.